docs/apache-airflow/faq.rst

```diff
@@ -522,3 +522,27 @@ This means ``explicit_defaults_for_timestamp`` is disabled in your mysql server
 
 #. Set ``explicit_defaults_for_timestamp = 1`` under the ``mysqld`` section in your ``my.cnf`` file.
 #. Restart the Mysql server.
+
+Does Airflow collect any telemetry data?
+----------------------------------------
+
+.. _airflow-telemetry-faq:
+
+Airflow integrates `Scarf <https://about.scarf.sh/>`__ to collect basic telemetry data during operation.
+This data assists Airflow maintainers in better understanding how Airflow is used.
+Insights gained from this telemetry are critical for prioritizing patches, minor releases, and
+security fixes. Additionally, this information supports key decisions related to the development road map.
+
```

Comment:
worth adding some info about where it goes and how they can view it?

Reply:
what about "right to forget" i.e. is there a mechanism for getting already-tracked data removed?

Reply:
Aren't such rights only for PII data collected?

Reply:
They can't view it, atleast not now, just PMCs